content/examples/enterprise-quickstart/config.yaml

```diff
@@ -0,0 +1,26 @@
+authenticate_service_url: https://authenticate.localhost.pomerium.io
+
+idp_provider: REPLACE_ME
+idp_client_id: REPLACE_ME
+idp_client_secret: REPLACE_ME
```

Comment:
`idp_provider_url: ` - compulsory element

Reply:
@ssveta7ak this depends on your IdP. GitHub, for instance, doesn't require that, nor does Google. So the IdP configuration will vary.